src/filesystem/simple_fs_nix.cpp

```diff
@@ -192,7 +192,7 @@ void restore_state(file_system& fs, native_string_view data) {
 namespace impl {
 bool contains_non_ascii(native_char const* str) {
 	for(auto c = str; *c != 0; ++c) {
-		if(int32_t(*c) > 127 || int32_t(*c) < 0)
```

Comment:
this was a protection against the possibility that `char` could be unsigned in some compilers